manim/mobject/geometry/arc.py

```diff
@@ -126,10 +135,10 @@ def add_tip(
     def create_tip(
         self,
         tip_shape: type[tips.ArrowTip] | None = None,
-        tip_length: float = None,
-        tip_width: float = None,
+        tip_length: float | None = None,
+        tip_width: float | None = None,
         at_start: bool = False,
-    ):
+    ) -> tips.ArrowTip | tips.ArrowTriangleFilledTip:
```

Comment:
This should probably just be `tips.ArrowTip`, because the extra union doesn't give us any more information.
```suggestion
    ) -> tips.ArrowTip:
```

Reply:
This works fine.